tests/network/l2_bridge/nad_ref_change/conftest.py

```
@@ -145,3 +147,73 @@ def baseline_connectivity(
+        vm.start(wait=True)
+        vm.wait_for_agent_connected()
+        wait_for_ifaces_status(
+            vm=vm,
+            ip_addresses_by_spec_net_name={
+                LINUX_BRIDGE_IFACE_NAME_1: [addr.split("/")[0] for addr in iface_a_ips],
+            },
+        )
+        yield vm
+
+
+@pytest.fixture()
```

Ruff (0.15.15)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)